  0%|          | 0/2 [00:00<?, ?it/s] 50%|█████     | 1/2 [00:03<00:03,  3.18s/it]100%|██████████| 2/2 [00:05<00:00,  2.95s/it]100%|██████████| 2/2 [00:05<00:00,  2.86s/it]
  0%|          | 0/3 [00:00<?, ?it/s] 33%|███▎      | 1/3 [00:02<00:05,  2.84s/it] 67%|██████▋   | 2/3 [00:05<00:02,  2.81s/it]100%|██████████| 3/3 [00:08<00:00,  2.74s/it]100%|██████████| 3/3 [00:08<00:00,  2.73s/it]
running: ./test_config.json
CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.launch --master_port 10027 --nproc_per_node=1 ./tools/relation_test_net.py --config-file "./configs/e2e_relation_X_101_32_8_FPN_1x.yaml" MODEL.ROI_RELATION_HEAD.USE_GT_BOX False MODEL.ROI_RELATION_HEAD.USE_GT_OBJECT_LABEL False MODEL.ROI_RELATION_HEAD.PREDICTOR CausalAnalysisPredictor MODEL.ROI_RELATION_HEAD.CAUSAL.EFFECT_TYPE TDE MODEL.ROI_RELATION_HEAD.CAUSAL.FUSION_TYPE sum  MODEL.ROI_RELATION_HEAD.CAUSAL.CONTEXT_LAYER motifs  TEST.IMS_PER_BATCH 1  DTYPE "float16"  GLOVE_DIR ./../../checkpoints/glove MODEL.PRETRAINED_DETECTOR_CKPT ./../../checkpoints/model OUTPUT_DIR ./../../checkpoints/model  TEST.CUSTUM_EVAL True TEST.CUSTUM_PATH ./../data/interim/catontv_ab-1.22.1c/train DETECTED_SGG_DIR ./../data/sg_processed/catontv_ab-1.22.1c/train > ./../data/sg_processed/catontv_ab-1.22.1c/train/run.log 
CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.launch --master_port 10027 --nproc_per_node=1 ./tools/relation_test_net.py --config-file "./configs/e2e_relation_X_101_32_8_FPN_1x.yaml" MODEL.ROI_RELATION_HEAD.USE_GT_BOX False MODEL.ROI_RELATION_HEAD.USE_GT_OBJECT_LABEL False MODEL.ROI_RELATION_HEAD.PREDICTOR CausalAnalysisPredictor MODEL.ROI_RELATION_HEAD.CAUSAL.EFFECT_TYPE TDE MODEL.ROI_RELATION_HEAD.CAUSAL.FUSION_TYPE sum  MODEL.ROI_RELATION_HEAD.CAUSAL.CONTEXT_LAYER motifs  TEST.IMS_PER_BATCH 1  DTYPE "float16"  GLOVE_DIR ./../../checkpoints/glove MODEL.PRETRAINED_DETECTOR_CKPT ./../../checkpoints/model OUTPUT_DIR ./../../checkpoints/model  TEST.CUSTUM_EVAL True TEST.CUSTUM_PATH ./../data/interim/catontv_ab-1.22.1c/test DETECTED_SGG_DIR ./../data/sg_processed/catontv_ab-1.22.1c/test > ./../data/sg_processed/catontv_ab-1.22.1c/test/run.log 
Traceback (most recent call last):
  File "test_main.py", line 45, in <module>
    fo_models = vdp.utils.get_sgg_fo_models(sg_input_dir, box_topk=fo_params['box_topk'], rel_topk=fo_params['rel_topk'], raw_img_dir=os.path.dirname(test_imgs[0]))
  File "/scratch/users/atharva2/vdp/vdp/utils.py", line 101, in get_sgg_fo_models
    img_path = image_paths[image_idx]
TypeError: list indices must be integers or slices, not str
